src/Uno.UI/UI/Xaml/UIElement.netstd.cs

```diff
@@ -196,5 +196,55 @@ private void OnChildRemoved(UIElement child)
 
 		internal Point GetPosition(Point position, UIElement relativeTo)
 			=> TransformToVisual(relativeTo).TransformPoint(position);
+
+#if DEBUG
+
+		/// <summary>
+		/// Convenience method to find all views with the given name.
+		/// </summary>
+		public FrameworkElement[] FindViewsByName(string name) => FindViewsByName(name, searchDescendantsOnly: false);
+
+
+		/// <summary>
+		/// Convenience method to find all views with the given name.
+		/// </summary>
+		/// <param name="searchDescendantsOnly">If true, only look in descendants of the current view; otherwise search the entire visual tree.</param>
+		public FrameworkElement[] FindViewsByName(string name, bool searchDescendantsOnly)
```

Comment:
`IEnumerable<FrameworkElement>` ?

Reply:
IEnumerable makes it more difficult to visualize in the debugger